css-structure/css-structure_selectors-and-cascading/css/styles.css

```diff
@@ -49,92 +49,94 @@ nav li {
 /* -------------------------------------------------------- */
 
 /* 1. These styles should apply to the headline */
-FIX_ME {
+header {
```

Comment:
Headline here means the first level title `<h1>`, not header. , I made a mistake here too)))